packages/venia-concept/src/RootComponents/Product/index.js

```diff
@@ -4,4 +4,4 @@
  * pageTypes = PRODUCT
```

Comment:
RootComponents should be aware of the Redux store. We've already got a `ProductFullDetail` abstraction going on, so I don't think that we need an extra container just for binding to Redux. The `Product` RootComponent's job is to "handle" the product data, and provide an implementation for expressing it. I think that the binding to Redux actions falls within its scope.